### ISSUE TITLE
Express checkout "Customize" button not entirely linked

### ISSUE THREAD

**Issue by aheckler** [Automattic/woocommerce-payments#6365]
Pretty nitpicky, but when I go to customize, say, Apple Pay and Google Pay, I have to click this, which seems at first glance to be a button:

![Screenshot taken on 2023-05-24 at 15 25 27 UTC@2x](https://github.com/Automattic/woocommerce-payments/assets/5512652/f812d112-2a02-4a15-b4ce-2915203b9427)

However, the link itself is only on the word `Customize`, not the entire button. So this is effectively a clickable link with some padding that looks like, but does not act like, a button! It's slightly confusing.

Can we just make the entire "button" clickable?

For example, I'd expect that clicking here would take me to the customization page, but it's not clickable.

![Screenshot taken on 2023-05-24 at 15 27 45 UTC@2x](https://github.com/Automattic/woocommerce-payments/assets/5512652/0b2c757e-0894-4b14-a2dc-64c22ed3afde)

The padding area where you cannot click is in green:

<img width="311" alt="Screenshot taken on 2023-05-24 at 15 28 27 UTC@2x" src="https://github.com/Automattic/woocommerce-payments/assets/5512652/2f00e1f2-4f50-449c-8a48-c84b78e23e1a">

**Comment by aheckler** [comment]
Fixed by https://github.com/Automattic/woocommerce-payments/pull/7198
